src/main/resources/com/mathworks/ci/tools/MatlabInstaller/help-products.html

```diff
@@ -0,0 +1,13 @@
+<div>
+  <p>
+    Insert a space-separated list of products to install. The plugin installs the specified products in addition to MATLAB.
+  </p>
+  <p>
+    For a list of supported products, open the input file for your preferred release from the <a href="https://github.com/mathworks-ref-arch/matlab-dockerfile/tree/main/mpm-input-files">mpm-input-files </a>folder on GitHub.
+    Specify products using the format shown in the input file, excluding the #product. prefix. For example, to install Deep Learning Toolbox™ in addition to MATLAB, insert Deep_Learning_Toolbox in the Products box.
```

Comment:
Suggest removing the trademark symbol if it's not used in other help elements.

Reply:
Done